client/tests/integration/unstable_network.rs

```diff
@@ -65,7 +65,7 @@ fn unstable_network(
     let (network, mut iroha_client) = rt.block_on(async {
         let mut configuration = Configuration::test();
         configuration.queue.maximum_transactions_in_block = MAXIMUM_TRANSACTIONS_IN_BLOCK;
-        configuration.logger.max_log_level = Level::ERROR.into();
+        configuration.logger.max_log_level = Level::TRACE.into();
```

Comment:
This is a safe default for most tests. If it's not written to `stdout` it gets dropped, so it shouldn't make things much slower.